docker/mysql_tests/scripts/tests/delete_end_to_end_test.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+set -e -x
+
+export WALE_S3_PREFIX=s3://mysqldeleteendtoendbucket
+export WALG_MYSQL_DATASOURCE_NAME=sbtest:@/sbtest
+export WALG_MYSQL_BINLOG_DST=${MYSQLDATA}
+
+echo "CREATE DATABASE sbtest;" > /etc/mysql/init.sql
+echo "CREATE USER sbtest@localhost;" >> /etc/mysql/init.sql
+echo "GRANT ALL PRIVILEGES ON *.* TO sbtest@localhost;" >> /etc/mysql/init.sql
+
+echo "[mysqld]" >> /etc/mysql/my.cnf
```

Comment:
Let's extract it to a separate file, which will be delivered to the container via Dockerfile COPY.